build.ps1

```diff
@@ -6,14 +6,26 @@ function exec([string] $cmd) {
     }
 }
 
+# running tests
+exec "dotnet run -p tools/CreateJsonSchema"
+exec "dotnet test test\docfx.Test"
+exec "dotnet test test\docfx.Test -c Release"
+
+# packing
+$featureBranchPrefix = "feature/"
+$branch = & {git rev-parse --abbrev-ref HEAD}
 $commitSha = & { git describe --always }
 $commitCount = & { git rev-list --count HEAD }
 $revision = $commitCount.ToString().PadLeft(5, '0')
-$version = "3.0.0-alpha-$revision-$commitSha"
 
-Remove-Item ./drop -Force -Recurse -ErrorAction Ignore
+if ($branch -eq "v3") {
+    $version = "3.0.0-beta-$revision-$commitSha"
+} elseif ($branch.StartsWith($featureBranchPrefix)) {
+    $feature = $branch.SubString($featureBranchPrefix.length)
+    $version = "3.0.0-alpha-$feature-$revision-$commitSha"
+} else {
+    exit 0
```

Comment:
use `3.0.0-alpha-$revision-$sha` for other branches to we could test pack locally on other branches?